assets/src/components/detours/diversionPage.tsx

```diff
@@ -233,6 +233,11 @@ export const DiversionPage = ({
               onNavigateBack={editDetour}
               detourText={textArea}
               onChangeDetourText={setTextArea}
+              onActivateDetour={
+                inTestGroup(TestGroups.DetoursList)
+                  ? () => alert("Activate!")
```

Comment:
nit: I can't quite articulate why, so I guess take this as non-blocking, but I'd prefer to not use `alert` if we don't really need it.

Reply:
Does knowing that this is only a temporary thing change your feeling?

Reply:
Slightly, I'm fine with it being non-blocking, but I'm inclined to enable the https://eslint.org/docs/latest/rules/no-alert rule

Reply:
Oh I'm happy to remove it - I used `alert` because it was an easy way to locally test it, but I'm not attached.

Reply:
[It's gone, btw](https://github.com/mbta/skate/pull/2713/commits/cec1abd6fbdbb76e57c60e4d4b11ed0fbe431cc1)